src/libdredd/src/mutation_replace_binary_operator.cc

```diff
@@ -346,16 +348,17 @@ void MutationReplaceBinaryOperator::GenerateBinaryOperatorReplacement(
     bool only_track_mutant_coverage, int mutation_id_base,
     std::stringstream& new_function, int& mutation_id_offset,
     protobufs::MutationReplaceBinaryOperator& protobuf_message) const {
-  for (auto operator_kind :
+  for (auto replacement_operator :
        GetReplacementOperators(optimise_mutations, ast_context)) {
     if (!only_track_mutant_coverage) {
-      new_function << "  if (__dredd_enabled_mutation(local_mutation_id + "
-                   << mutation_id_offset << ")) return " << arg1_evaluated
-                   << " "
-                   << clang::BinaryOperator::getOpcodeStr(operator_kind).str()
-                   << " " << arg2_evaluated << ";\n";
+      new_function
+          << "  if (__dredd_enabled_mutation(local_mutation_id + "
+          << mutation_id_offset << ")) return " << arg1_evaluated << " "
```

Comment:
Not obvious to me what has changed in the above lines, so unsure why the formatting has changed but I assume clang-format has done this, perhaps due to the line below changing. No action needed assuming this is expected.